internal/ir/builder.go

```diff
@@ -531,7 +531,7 @@ func (b *builder) lookup(fn *Function, name string) Value {
 	}
 
 	if global, ok := fn.File.Members[name].(*Global); ok {
-		return global
+		return fn.addLocal(b.expr(fn, global.Value, false), global.Value)
```

Comment:
Since now we are always returning a new variable with the global value we can remove this duplicated logic from [builder.identAssing](https://github.com/ZupIT/horusec-engine/blob/semantic-engine/internal/ir/builder.go#L508)

Reply:
Also add `/* expand */` after false to more easily readable.